plugins/janus_recordplay.c

```diff
@@ -2618,6 +2677,74 @@ static void *janus_recordplay_playout_thread(void *data) {
 				}
 			}
 		}
+		if(data) {
+			if(data == session->dframes) {
+				/* First packet, send now */
+				/* Data recording stores raw UDP packets */
```

Comment:
Corrected to extract recording timestamp. For the timebeing, the recording timestamp is not used, since the frame timestamp (data->ts) can be used directly, as it follows the recording timestamp.
It is always: recording_timestamp(current_frame) = timestamp(current_frame) + recording_timestamp(initial_frame).